script.js

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable quotes */
+/* eslint-disable no-unused-vars */
+import _ from "lodash";
+import "./style.css";
+import "./CRUD.js";
+import { refresh } from "./complete.js";
+
+refresh();
+
+const notifyElement = document.querySelector(".notify");
+function showNotification(msg) {
+  notifyElement.innerHTML = msg;
+
+  notifyElement.classList.add("notified");
+
+  setTimeout(() => {
+    notifyElement.classList.remove("notified");
+  }, 2000);
+}
+
+/* eslint-disable indent */
+/* eslint-disable comma-dangle */
+/* eslint-disable quotes */
+/* eslint-disable no-bitwise */
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-expressions */
+// select element
+
+const form = document.getElementById("todoform");
+const todoInput = document.getElementById("newtodo");
+const todoListElement = document.getElementById("tododslist");
+const clean = document.getElementById("reset");
+
+// My variables
+let todos = JSON.parse(localStorage.getItem("todos")) || [];
+let editToDoId = -1;
+
+// First render
+renderTodos();
+clearCompleted(todoListElement);
+
+// To submit form
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+
+  saveTodo();
+  renderTodos();
+  localStorage.setItem("todos", JSON.stringify(todos));
+});
+
+// to save to do
+function saveTodo() {
+  const todoValue = todoInput.value;
+
+  // check if empty
+  const isEmpty = todoValue === "";
+
+  //  check for duplicates
+  const isDuplicate = todos.some(
+    (todo) => todo.value.toUpperCase() === todoValue.toUpperCase()
+  );
+  if (isEmpty) {
+    alert("Todo's input is empty");
```

Comment:
According to JS best practices, we must avoid using Alerts as the block the code run, Instead we can use customized error messages or modal view